judo-runtime-core-dispatcher/src/main/java/hu/blackbelt/judo/runtime/core/dispatcher/behaviours/QueryCustomizerParameterProcessor.java

```diff
@@ -205,6 +205,7 @@ private DAO.Seek extractSeekParameter(final Map<String, Object> queryCustomizerP
 			final Map<String, Object> _seek = (Map<String, Object>) queryCustomizerParameter.get("_seek");
             return DAO.Seek.builder()
                     .limit((Integer) _seek.get("limit"))
+                    .offset(_seek.get("offset") != null ? (Integer) _seek.get("offset") : -1)
```

Comment:
why do we default to `-1`?

Reply:
Because -1 is a value which use to be used when a positive integer / long is not set.